src/app/cube/details/components/action-panel/editorial-action-pad/editorial-action-pad.component.ts

```diff
@@ -82,7 +84,7 @@ export class EditorialActionPadComponent implements OnInit {
   async createRevision() {
     this.closeRevisionModal();
     this.toaster.success('One Moment Please', 'Your revision is being created.');
-    await this.learningObjectService
+    await this.revisionsService
       .createRevision(this.learningObject.cuid).then(async (revisionUri: any) => {
         this.revisedLearningObject = (await this.learningObjectServiceUri.fetchUri(revisionUri.revisionUri).toPromise())[0];
         this.router.navigate([`/onion/learning-object-builder/${this.revisedLearningObject.cuid}`]);
```

Comment:
We may need to add the version as a query or path param to the router.navigate because cuid is shared across versions.

Reply:
+1